[
    {
        "cve": "CVE-2024-5480",
        "created_datetime": "2024-06-08T10:53:01",
        "author_id": "303126470",
        "data_id": "1799394193418379273",
        "reposts": 0,
        "post": "pytorch 2.3.1 \u30ea\u30ea\u30fc\u30b9\u3001\u30d0\u30b0\u4fee\u6b63\u30ea\u30ea\u30fc\u30b9\npytorch \u5206\u6563 rpc \u30d5\u30ec\u30fc\u30e0\u30ef\u30fc\u30af\u306b\u91cd\u5927\u306a\u8106\u5f31\u6027 ( cve-2024-5480 cvss10 critical )\n\u5bfe\u7b56\uff1a\u30a2\u30c3\u30d7\u30c7\u30fc\u30c8\u306f\u3053\u3061\u3089\ud83d\udc49 https://t.co/uyhkx2y4yk",
        "audience": 2039,
        "audience_total": 2869,
        "total_reposts": 3,
        "post_count": 3,
        "author": "\u4e8c\u672c\u677e\u54f2\u4e5f",
        "username": "t_nihonmatsu",
        "v_score": 0.326299,
        "e_score": 0.00043,
        "base_score": "10",
        "description": "A VULNERABILITY IN THE PYTORCHS TORCH.DISTRIBUTED.RPC FRAMEWORK, SPECIFICALLY IN VERSIONS PRIOR TO 2.2.2, ALLOWS FOR REMOTE CODE EXECUTION (RCE). THE FRAMEWORK, WHICH IS USED IN DISTRIBUTED TRAINING SCENARIOS, DOES NOT PROPERLY VERIFY THE FUNCTIONS BEING CALLED DURING RPC (REMOTE PROCEDURE CALL) OPERATIONS. THIS OVERSIGHT PERMITS ATTACKERS TO EXECUTE ARBITRARY COMMANDS BY LEVERAGING BUILT-IN PYTHON FUNCTIONS SUCH AS EVAL DURING MULTI-CPU RPC COMMUNICATION. THE VULNERABILITY ARISES FROM THE LACK OF RESTRICTION ON FUNCTION CALLS WHEN A WORKER NODE SERIALIZES AND SENDS A PYTHONUDF (USER DEFINED FUNCTION) TO THE MASTER NODE, WHICH THEN DESERIALIZES AND EXECUTES THE FUNCTION WITHOUT VALIDATION. THIS FLAW CAN BE EXPLOITED TO COMPROMISE MASTER NODES INITIATING DISTRIBUTED TRAINING, POTENTIALLY LEADING TO THE THEFT OF SENSITIVE AI-RELATED DATA.",
        "last_modified_datetime": "2024-06-07 14:56:00",
        "published_datetime": "2024-06-06 19:16:00",
        "vendor_logo": null,
        "profile_url": "https://pbs.twimg.com/profile_images/1437738711756595200/xY4ZPYD6_normal.jpg",
        "feed_name": "twitter"
    },
    {
        "cve": "CVE-2024-5480",
        "created_datetime": "2024-06-08T14:05:03",
        "author_id": "1432098253521162243",
        "data_id": "1799442520365015415",
        "reposts": 1,
        "post": "rt @dinosn: cve-2024-5480 (cvss 10): critical rce vulnerability in pytorch distributed rpc framework https://t.co/cdubntom8p",
        "audience": 174,
        "audience_total": 2869,
        "total_reposts": 3,
        "post_count": 3,
        "author": "north4",
        "username": "north4tea",
        "v_score": 0.326299,
        "e_score": 0.00043,
        "base_score": "10",
        "description": "A VULNERABILITY IN THE PYTORCHS TORCH.DISTRIBUTED.RPC FRAMEWORK, SPECIFICALLY IN VERSIONS PRIOR TO 2.2.2, ALLOWS FOR REMOTE CODE EXECUTION (RCE). THE FRAMEWORK, WHICH IS USED IN DISTRIBUTED TRAINING SCENARIOS, DOES NOT PROPERLY VERIFY THE FUNCTIONS BEING CALLED DURING RPC (REMOTE PROCEDURE CALL) OPERATIONS. THIS OVERSIGHT PERMITS ATTACKERS TO EXECUTE ARBITRARY COMMANDS BY LEVERAGING BUILT-IN PYTHON FUNCTIONS SUCH AS EVAL DURING MULTI-CPU RPC COMMUNICATION. THE VULNERABILITY ARISES FROM THE LACK OF RESTRICTION ON FUNCTION CALLS WHEN A WORKER NODE SERIALIZES AND SENDS A PYTHONUDF (USER DEFINED FUNCTION) TO THE MASTER NODE, WHICH THEN DESERIALIZES AND EXECUTES THE FUNCTION WITHOUT VALIDATION. THIS FLAW CAN BE EXPLOITED TO COMPROMISE MASTER NODES INITIATING DISTRIBUTED TRAINING, POTENTIALLY LEADING TO THE THEFT OF SENSITIVE AI-RELATED DATA.",
        "last_modified_datetime": "2024-06-07 14:56:00",
        "published_datetime": "2024-06-06 19:16:00",
        "vendor_logo": null,
        "profile_url": null,
        "feed_name": "twitter"
    },
    {
        "cve": "CVE-2024-5480",
        "created_datetime": "2024-06-09T05:05:50",
        "author_id": "1756999699679338496",
        "data_id": "1799669208462295315",
        "reposts": 0,
        "post": "critical : cvss 10 : attn!! artificial intelligence(ai) companies and researchers using pytorch.\nmitigate cve-2024-5480 asap https://t.co/sgog8tzwvv",
        "audience": 456,
        "audience_total": 2869,
        "total_reposts": 3,
        "post_count": 3,
        "author": "patch now !!",
        "username": "patchnow24x7",
        "v_score": 0.326299,
        "e_score": 0.00043,
        "base_score": "10",
        "description": "A VULNERABILITY IN THE PYTORCHS TORCH.DISTRIBUTED.RPC FRAMEWORK, SPECIFICALLY IN VERSIONS PRIOR TO 2.2.2, ALLOWS FOR REMOTE CODE EXECUTION (RCE). THE FRAMEWORK, WHICH IS USED IN DISTRIBUTED TRAINING SCENARIOS, DOES NOT PROPERLY VERIFY THE FUNCTIONS BEING CALLED DURING RPC (REMOTE PROCEDURE CALL) OPERATIONS. THIS OVERSIGHT PERMITS ATTACKERS TO EXECUTE ARBITRARY COMMANDS BY LEVERAGING BUILT-IN PYTHON FUNCTIONS SUCH AS EVAL DURING MULTI-CPU RPC COMMUNICATION. THE VULNERABILITY ARISES FROM THE LACK OF RESTRICTION ON FUNCTION CALLS WHEN A WORKER NODE SERIALIZES AND SENDS A PYTHONUDF (USER DEFINED FUNCTION) TO THE MASTER NODE, WHICH THEN DESERIALIZES AND EXECUTES THE FUNCTION WITHOUT VALIDATION. THIS FLAW CAN BE EXPLOITED TO COMPROMISE MASTER NODES INITIATING DISTRIBUTED TRAINING, POTENTIALLY LEADING TO THE THEFT OF SENSITIVE AI-RELATED DATA.",
        "last_modified_datetime": "2024-06-07 14:56:00",
        "published_datetime": "2024-06-06 19:16:00",
        "vendor_logo": null,
        "profile_url": null,
        "feed_name": "twitter"
    },
    {
        "cve": "CVE-2024-5480",
        "created_datetime": "2024-06-09T05:06:02",
        "author_id": "1756999699679338496",
        "data_id": "1799669260995928356",
        "reposts": 0,
        "post": "critical : cvss 10 : attn!! artificial intelligence(ai) companies and researchers using pytorch.\nmitigate cve-2024-5480 asap https://t.co/sgog8tzwvv",
        "audience": 456,
        "audience_total": 2869,
        "total_reposts": 3,
        "post_count": 3,
        "author": "patch now !!",
        "username": "patchnow24x7",
        "v_score": 0.326299,
        "e_score": 0.00043,
        "base_score": "10",
        "description": "A VULNERABILITY IN THE PYTORCHS TORCH.DISTRIBUTED.RPC FRAMEWORK, SPECIFICALLY IN VERSIONS PRIOR TO 2.2.2, ALLOWS FOR REMOTE CODE EXECUTION (RCE). THE FRAMEWORK, WHICH IS USED IN DISTRIBUTED TRAINING SCENARIOS, DOES NOT PROPERLY VERIFY THE FUNCTIONS BEING CALLED DURING RPC (REMOTE PROCEDURE CALL) OPERATIONS. THIS OVERSIGHT PERMITS ATTACKERS TO EXECUTE ARBITRARY COMMANDS BY LEVERAGING BUILT-IN PYTHON FUNCTIONS SUCH AS EVAL DURING MULTI-CPU RPC COMMUNICATION. THE VULNERABILITY ARISES FROM THE LACK OF RESTRICTION ON FUNCTION CALLS WHEN A WORKER NODE SERIALIZES AND SENDS A PYTHONUDF (USER DEFINED FUNCTION) TO THE MASTER NODE, WHICH THEN DESERIALIZES AND EXECUTES THE FUNCTION WITHOUT VALIDATION. THIS FLAW CAN BE EXPLOITED TO COMPROMISE MASTER NODES INITIATING DISTRIBUTED TRAINING, POTENTIALLY LEADING TO THE THEFT OF SENSITIVE AI-RELATED DATA.",
        "last_modified_datetime": "2024-06-07 14:56:00",
        "published_datetime": "2024-06-06 19:16:00",
        "vendor_logo": null,
        "profile_url": null,
        "feed_name": "twitter"
    },
    {
        "cve": "CVE-2024-5480",
        "created_datetime": "2024-06-09T05:09:33",
        "author_id": "T2_10JZ5IQI4R",
        "data_id": "1DBM14W",
        "reposts": 2,
        "post": "critical : cvss 10 : attn!! artificial intelligence(ai) companies and researchers using pytorch.\nmitigate cve-2024-5480",
        "audience": 200,
        "audience_total": 2869,
        "total_reposts": 3,
        "post_count": 3,
        "author": "patchnow24x7",
        "username": "t3_1dbm14w",
        "v_score": 0.326299,
        "e_score": 0.00043,
        "base_score": "10",
        "description": "A VULNERABILITY IN THE PYTORCHS TORCH.DISTRIBUTED.RPC FRAMEWORK, SPECIFICALLY IN VERSIONS PRIOR TO 2.2.2, ALLOWS FOR REMOTE CODE EXECUTION (RCE). THE FRAMEWORK, WHICH IS USED IN DISTRIBUTED TRAINING SCENARIOS, DOES NOT PROPERLY VERIFY THE FUNCTIONS BEING CALLED DURING RPC (REMOTE PROCEDURE CALL) OPERATIONS. THIS OVERSIGHT PERMITS ATTACKERS TO EXECUTE ARBITRARY COMMANDS BY LEVERAGING BUILT-IN PYTHON FUNCTIONS SUCH AS EVAL DURING MULTI-CPU RPC COMMUNICATION. THE VULNERABILITY ARISES FROM THE LACK OF RESTRICTION ON FUNCTION CALLS WHEN A WORKER NODE SERIALIZES AND SENDS A PYTHONUDF (USER DEFINED FUNCTION) TO THE MASTER NODE, WHICH THEN DESERIALIZES AND EXECUTES THE FUNCTION WITHOUT VALIDATION. THIS FLAW CAN BE EXPLOITED TO COMPROMISE MASTER NODES INITIATING DISTRIBUTED TRAINING, POTENTIALLY LEADING TO THE THEFT OF SENSITIVE AI-RELATED DATA.",
        "last_modified_datetime": "2024-06-07 14:56:00",
        "published_datetime": "2024-06-06 19:16:00",
        "vendor_logo": null,
        "profile_url": null,
        "feed_name": "reddit"
    }
]